packages/wrangler/src/publish/publish.ts

```diff
@@ -923,16 +925,20 @@ function isAuthenticationError(e: unknown): e is ParseError {
 }
 
 async function ensureQueuesExist(config: Config) {
-	const producers = (config.queues.producers || []).map((x) => x.queue);
-	const consumers = (config.queues.consumers || []).map((x) => x.queue);
+	const producers = (config.queues.producers || []).map(
+		(producer) => producer.queue
+	);
+	const consumers = (config.queues.consumers || []).map(
+		(consumer) => consumer.queue
+	);
 
 	const queueNames = producers.concat(consumers);
 	for (const queue of queueNames) {
 		try {
 			await getQueue(config, queue);
 		} catch (err) {
-			const x = err as FetchError;
-			if (x.code === 100123) {
+			const queueErr = err as FetchError;
+			if (queueErr.code === 100123) {
 				// queue_not_found
 				throw new Error(
 					`Queue "${queue}" does not exist. To create it, run: wrangler queues create ${queue}`
```

Comment:
I missed it in the first review, but this `catch` block shouldn't swallow all non-handled errors—it should re-throw them

Reply:
Whoops, that was the intent. Nice catch! Fixed.